docs/tutorials/developer-journey/level-2/2.4-intro-candid.md

```diff
@@ -0,0 +1,305 @@
+# 2.4: Introduction to Candid 
+
+## Overview
+
+Candid is an interface description language that is used to describe the public interface of a service. A **service** is usually in the form of an application deployed as a canister on the IC. Public interfaces of canisters are used to interact with the canister. Candid supports interactions through the IC SDK via the terminal, through a web browser, or through the use of agents. It also provides a way to specify input argument values and display return values from different canister **methods**, regardless of the manner used to interact with the canister. Recall that a method is a piece of code specifying a task which declares a sequence of arguments and their associated result types.
+
+Candid is language-agnostic, which is a key benefit since it allows for the inter-operation between backend canisters and frontend canisters written in different languages, such as Motoko, Rust, and JavasScript. 
```

Comment:
I'd rather say between canisters and client applications.

A "frontend canister" doesn't interact with a backend canister. The browser fetches the assets from the frontend canister, and the application running in the browser calls the backend canister.